src/sos-packaging.props

```diff
@@ -1,4 +1,9 @@
-﻿<Project>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Are we Apache license?

Reply:
I copied that from Directory.Build.props.  Maybe that and other files are using the wrong copyright notice.

Reply:
It looks like most of the diagnostics repo's props, target and csproj files use this license.

Reply:
We might need to fix this. I don't think we have any dependency that requires us to use this license. Our packages are listed as MIT licensed and so are coreclr and corefx.